src/Constants.js

```diff
@@ -312,6 +342,7 @@ export const ZORAS_RIVER = [
     "Zora-River-Lower-Freestanding-PoH",
     "Zora-River-Upper-Freestanding-PoH",
     "Zora-River-Plateau-Open-Grotto-Chest",
+    "Magic-Bean-Salesman",
```

Comment:
Beans don't get randomized because bean logic would be terrible and would have to assume that you buy all 10 before requiring any bean location item